projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.ts

```diff
@@ -20,7 +27,7 @@ import { PlusMenuPosition } from './plus-menu-position.enum';
   encapsulation: ViewEncapsulation.None
 })
 export class PlusMenuComponent {
-  @Input() items = [];
+  @Input() items: PlusMenuItem[] = [];
   @Input() position = PlusMenuPosition.Right;
```

Comment:
sorry, i meant typing all inputs

Reply:
Like @Input() menuColor = '#9fce36' not sure we need to add obvious types like that.